### PR TITLE
Merge new 2019 Civic and existing 2017-18 Civic Hatch

### PR DESCRIPTION
**Pull request by csouers**
Hi. I have a user in Canada with a 2019 Civic Sedan Touring that's now working. Out of the box, it fingerprinted as 2017 hatchback. Only difference I've noticed in this car is a lack of DOORS_STATUS at 0x405 (still looking for this if it moved). Right now, we're using self.standstill and self.door_all_closed like on Accord as the only major changes. Only remaining complaint is improper reporting of set speed as he uses Metric on the EON (to be expected AFAIK).

As both cars will fingerprint as Hatchback EX, we could potentially merge these. We have a user with hatchback touring if we want to replace the EX print with that. Vehicle weights are obviously slightly different. DBC file is unchanged as it used as-is from the 2017 car. Note: early stages of testing on this, but initial impressions are its working fine.

### CODE REVIEW

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -91,8 +91,13 @@ class CAR:
...
   }],
-  CAR.CIVIC_HATCH: [{
+  CAR.CIVIC_BOSCH: [{
+    #2017 Civic Hatchback EX
     57: 3, 148: 8, 228: 5, 304: 8, 330: 8, 344: 8, 380: 8, 399: 7, 401: 8, 420: 8, 427: 3, 428: 8, 432: 7, 441: 5, 450: 8, 464: 8, 470: 2, 476: 7, 477: 8, 479: 8, 490: 8, 493: 5, 495: 8, 506: 8, 545: 6, 597: 8, 662: 4, 773: 7, 777: 8, 780: 8, 795: 8, 800: 8, 804: 8, 806: 8, 808: 8, 829: 5, 862: 8, 884: 8, 891: 8, 892: 8, 927: 8, 929: 8, 985: 3, 1024: 5, 1027: 5, 1029: 8, 1036: 8, 1039: 8, 1108: 8, 1302: 8, 1322: 5, 1361: 5, 1365: 5, 1424: 5, 1600: 5, 1601: 8, 1633: 8
+  },
+    #2019 Civic Sedan Touring Canadian
+  {
+    57: 3, 148: 8, 228: 5, 304: 8, 330: 8, 344: 8, 380: 8, 399:7, 401: 8, 420: 8, 427: 3, 428: 8, 432: 7, 441: 5, 450: 8, 464: 8, 470:2, 476: 7, 477: 8, 479: 8, 490: 8, 495: 8, 506: 8, 545: 6, 597: 8, 662:4, 773: 7, 777: 8, 780: 8, 795: 8, 804: 8, 806: 8, 808: 8, 829: 5, 862:8, 884: 8, 891: 8, 927: 8, 929: 8, 985: 3, 1302: 8, 1361: 5, 1365: 5, 1424: 5, 1600: 5, 1601: 8
```
Comment (rbiasini) [inline selfdrive/car/honda/values.py]: `399:7`, missing space

Comment (csouers) [reply selfdrive/car/honda/values.py]: Fixed.

**Comment by rbiasini** [comment]
looks good @csouers . before merging, maybe better to consider the civic '19 a separate model, as tehre seem to be dbc changes

**Comment by csouers** [comment]
Ok. We'll re-pull fingerprints and do some more testing.

**Comment by rbiasini** [comment]
@csouers, cool. Let me know when you have a chance to test. Would love to merge this.

**Comment by rbiasini** [comment]
@csouers, is this good to go?

**Comment by csouers** [comment]
Negative. Still working with two users on this. Will update again here when it is.

**Comment by csouers** [comment]
Worked on this again tonight with our 2019 Civic user (our 2018 Hatchback Touring user no longer has his car). 

Fingerprints didn't change. We tested splitting the cars. The new civic never fingerprints (more than 1 choice; it's also in the hatchback fingerprint). Hatchback always fingerprinted fine because it has frames that the new car doesn't. Any idea on this or should we just merge as per this PR?

I'm waiting to verify lower speed dropout on these new cars ~3-4 mph. Will check the drive once I have it and update here. Hopefully tomorrow.

File: selfdrive/car/honda/values.py [hunk 2]
```diff
@@ -91,8 +91,13 @@ class CAR:
   CAR.CIVIC: [{
     57: 3, 148: 8, 228: 5, 304: 8, 330: 8, 344: 8, 380: 8, 399: 7, 401: 8, 420: 8, 427: 3, 428: 8, 432: 7, 450: 8, 464: 8, 470: 2, 476: 7, 487: 4, 490: 8, 493: 5, 506: 8, 512: 6, 513: 6, 545: 6, 597: 8, 662: 4, 773: 7, 777: 8, 780: 8, 795: 8, 800: 8, 804: 8, 806: 8, 808: 8, 829: 5, 862: 8, 884: 8, 891: 8, 892: 8, 927: 8, 929: 8, 985: 3, 1024: 5, 1027: 5, 1029: 8, 1036: 8, 1039: 8, 1108: 8, 1302: 8, 1322: 5, 1361: 5, 1365: 5, 1424: 5, 1633: 8,
   }],
-  CAR.CIVIC_HATCH: [{
+  CAR.CIVIC_BOSCH: [{
+    #2017 Civic Hatchback EX
     57: 3, 148: 8, 228: 5, 304: 8, 330: 8, 344: 8, 380: 8, 399: 7, 401: 8, 420: 8, 427: 3, 428: 8, 432: 7, 441: 5, 450: 8, 464: 8, 470: 2, 476: 7, 477: 8, 479: 8, 490: 8, 493: 5, 495: 8, 506: 8, 545: 6, 597: 8, 662: 4, 773: 7, 777: 8, 780: 8, 795: 8, 800: 8, 804: 8, 806: 8, 808: 8, 829: 5, 862: 8, 884: 8, 891: 8, 892: 8, 927: 8, 929: 8, 985: 3, 1024: 5, 1027: 5, 1029: 8, 1036: 8, 1039: 8, 1108: 8, 1302: 8, 1322: 5, 1361: 5, 1365: 5, 1424: 5, 1600: 5, 1601: 8, 1633: 8
+  },
+    #2019 Civic Sedan Touring Canadian
```
[the remark this answers is not in the document]
Comment (csouers) [reply selfdrive/car/honda/values.py]: Combined and removed on latest commit. Should be good to go.

**Comment by rbiasini** [comment]
looks good. thanks @csouers

**Pull request merged**